src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -15,92 +21,67 @@
     @Test
     public void parse_allFieldsPresent_success() {
         Exercise expectedExercise = new ExerciseBuilder(PUSH_UP).build();
+        String expectedMessage = String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE);
 
-        assertParseSuccess(parser, " n/Push Up d/Test 1 at/09-10-2020 c/12345", new AddCommand(expectedExercise));
-
-        // whitespace only preamble
-        /*assertParseSuccess(parser, PREAMBLE_WHITESPACE + NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
-                + ADDRESS_DESC_BOB + TAG_DESC_FRIEND, new AddCommand(expectedExercise));
+        // normal input
+        assertParseSuccess(parser, " n/Push Up d/Test 1 at/09-10-2020 c/12345",
```

Comment:
Better to use variables instead of hardcoding

Eg. from AB3:
assertParseFailure(parser, "1 some random string", MESSAGE_INVALID_FORMAT);